src/main/java/com/spring/intro/controllers/HelloController.java

```diff
@@ -0,0 +1,22 @@
+package com.spring.intro.controllers;
+
+import com.spring.intro.dto.UserResponseDto;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class HelloController {
```

Comment:
Also this class does'n belong to the current h\w

Reply:
agree